docs/4.0/components/index.html

```diff
@@ -1,38 +1,11 @@
-<!doctype html>
-<html lang="en">
-  <head>
-    <meta charset="utf-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
-    <title>Bootstrap · Content moved</title>
-    <link rel="canonical" href="https://getbootstrap.com/docs/4.0/components/buttons/">
-    <meta http-equiv="refresh" content="0; url=https://getbootstrap.com/docs/4.0/components/buttons/">
-    <meta name="robots" content="noindex">
-    <style>
-      html {
-        display: flex;
-        align-items: center;
-        justify-content: center;
-        margin: 0;
-        width: 100vw;
-        height: 100vh;
-        text-align: center;
-      }
-      body {
-        font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Helvetica, Arial, sans-serif, "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol";
-      }
-      h1 {
-        margin-top: 0;
-        margin-bottom: .5rem;
-      }
-      a {
-        color: #007bff;
-        text-decoration: none;
-      }
-    </style>
-  </head>
-  <body>
-    <h1>Redirecting&hellip;</h1>
-    <a href="https://getbootstrap.com/docs/4.0/components/buttons/">Click here if you are not redirected</a>
-    <script>window.location="https://getbootstrap.com/docs/4.0/components/buttons/";</script>
-  </body>
+<!DOCTYPE html>
+<html lang="en-US">
+  <meta charset="utf-8">
+  <title>Redirecting&hellip;</title>
+  <link rel="canonical" href="https://getbootstrap.com/docs/4.0/components/buttons/">
```

Comment:
Do we need to move these `canonical` links to point to 4.2? I recall thinking about these as something to do with how we version our docs.